crates/runtime/src/dialogue.rs

```diff
@@ -138,6 +140,7 @@ fn visited_count(storage: Box<dyn VariableStorage>) -> yarn_fn_type! { impl Fn(S
     }
 }
 
+/*
```

Comment:
Why are all iterators commented out? :thinking:

Reply:
Bleh. Sorry. I meant to ask you about that and forgot. This naive implementation doesn't work when the bevy feature is active because `continue_` needs access to the `&mut World`. This functionality only seems to be used in tests, is it really needed?  I think it's possible to implement an iterator that holds the world reference but the api would change to something like `Dialogue.iter(&mut world)` which isn't as convenient. The other option would be to just skip this when the bevy feature is active.

Reply:
I'm fine with removing it entirely :+1:

Reply:
I tried out a couple of options here and ended up just having the code call `continue_` directly which feels a bit weird, mostly just because of the name of the function. It looks like it should be completely internal. Maybe we can rename it ot `continue`?

Reply:
`continue` is a keyword in rust, so you would rave to call it with `r#continue`, which is really awkward :sweat_smile: 
But if you have another suggestion, I have nothing against a rename :)

Reply:
Huh. I guess `continue_` is fine then.